src/ui/public/utils/mapping_setup.js

```diff
@@ -76,6 +76,7 @@ export default  function MappingSetupService(kbnIndex, esAdmin) {
     const prom = getKnownKibanaTypes()
     .then(function (knownTypes) {
       // if the type is in the knownTypes array already
+      return false;
```

Comment:
@spalger, this is a hack until your PR to remove client-side index pattern maintenance.